services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeCompletionServiceITest.kt

```diff
@@ -367,6 +369,72 @@ class HankeCompletionServiceITest(
         }
     }
 
+    @Nested
+    inner class IdsForDraftsToComplete {
+        /*
```

Comment:
There are some commented-out code here that I missed on first glance.

Reply:
Thank you for noticing. I removed the comment.